Modules/Core/Common/include/itkImage.h

```diff
@@ -189,6 +189,16 @@ class ITK_TEMPLATE_EXPORT Image : public ImageBase<VImageDimension>
   void
   Allocate(bool initializePixels = false) override;
 
+  /** Creates an image at the specified region. Allocates its pixel buffer, zero-initializing its pixels. */
```

Comment:
```suggestion
  /** Creates an image with the specified region. Allocates its pixel buffer, zero-initializing its pixels. */
```

Reply:
Thanks @dzenanz, that's fine _with_ me  😃

Reply:
Addressed "with" [this force-push](https://github.com/InsightSoftwareConsortium/ITK/compare/519ab58942d4c2baafc03a68ae74d561a5e36d30..a8ac2f1edb5b2a0ceb0e5a356de87bb505b4766b)